apps/web-giddh/src/app/invoice/preview/invoice.preview.component.ts

```diff
@@ -1408,4 +1434,61 @@ public toggleBulkUpdatePopup(isClose: boolean): void {
         return item;
     }
 
+    /**
+     * To adjustinvoice with advance receipts
+     *
+     * @param {ReceiptItem} item invoice details object
+     * @memberof InvoicePreviewComponent
+     */
+    public onPerformAdjustPaymentAction(item: ReceiptItem): void {
+        this.invFormData.voucherDetails.balanceDue = item.balanceDue.amountForAccount;
+        this.invFormData.voucherDetails.grandTotal = item.grandTotal.amountForAccount;
+        this.invFormData.voucherDetails.customerName = item.account.name;
+        this.invFormData.voucherDetails.customerUniquename = item.account.uniqueName;
+        this.invFormData.voucherDetails.voucherDate = item.voucherDate
+        this.invFormData.accountDetails.currencySymbol = item.accountCurrencySymbol;
+        this.changeStatusInvoiceUniqueName = item.uniqueName;
+        // To clear receipts voucher store
+        this.store.dispatch(this.invoiceReceiptActions.ResetVoucherDetails());
+        // To get re-assign receipts voucher store
+        this.store.dispatch(this.invoiceReceiptActions.getVoucherDetailsV4(item.account.uniqueName, {
+            invoiceNumber: item.voucherNumber,
+            voucherType: VoucherTypeEnum.sales
+        }));
+    }
+
+    /**
+        * To close advance reciipt modal
```

Comment:
Correct formatting of JS doc and reciipt spelling

Reply:
done